speech/google/cloud/speech/transcript.py

```diff
@@ -38,7 +38,7 @@ def from_api_repr(cls, transcript):
         :rtype: :class:`Transcript`
         :returns: Instance of ``Transcript``.
         """
-        return cls(transcript['transcript'], transcript['confidence'])
+        return cls(transcript.get('transcript'), transcript.get('confidence'))
```

Comment:
Can the `transcript` key actually be missing? If so, you need to add a test for that case.